Rails/Forms/odin_project_forms/app/views/users/new.html.erb

```diff
@@ -0,0 +1,12 @@
+<%= form_with(model: @user, local: true) do |f| %>
```

Comment:
To make things more compact, you can put the labels in the same line as the text fields -- e.g.

`Enter a username: <%= f.text_field :username %><br><br>`

That way you don't need to have label and text_field in 2 separate lines!
More on this at https://apidock.com/rails/ActionView/Helpers/FormHelper/form_for 😄

Reply:
That's pretty cool

Reply:
Good call! For some reason, I thought that merging the username line together would end up being too long - I looked into this and the [limit is 120 characters per line](https://shopify.github.io/ruby-style-guide/#:~:text=Limit%20lines%20to%20120%20characters,Avoid%20trailing%20whitespace.). It ended up being 113 characters, just below the limit!

Reply:
Oh I was confused by this. I thought the article was implying that adding the text like 

`Enter a username: <%= f.text_field :username %><br><br>`

Would create the label element and point it to the correct input, but it looks like it just renders the text in the same place, but without the `for="input_id"`. I feel like 

`Enter a username: <%= f.text_field :username %><br><br>` looks clean, but it misses the required `for` attribute.

Not sure if having two erb elements on the same line otherwise makes it easier to read.